test-support/src/main/scala/org/neo4j/spark/SparkConnectorScalaSuiteWithApocIT.scala

```diff
@@ -11,9 +11,12 @@ object SparkConnectorScalaSuiteWithApocIT {
   val server: Neo4jContainerExtension = new Neo4jContainerExtension()
     .withNeo4jConfig("dbms.security.auth_enabled", "false")
     .withEnv("NEO4J_ACCEPT_LICENSE_AGREEMENT", "yes")
-    .withEnv("NEO4JLABS_PLUGINS", "[\"apoc\"]")
     .withDatabases(Seq("db1", "db2"))
 
+  if (!TestUtil.experimental()) {
```

Comment:
please restore as it wase and add an assume in the `setUpContainer` method

Reply:
It doesn't work because the container is already started at line 11, trying to lead the non existing APOC

Reply:
But if I do this way the tests seem to fail https://github.com/neo4j-contrib/neo4j-spark-connector/runs/2772035901?check_suite_focus=true